### PR TITLE
[opt](memory) Optimize `ClearCacheAction`implementation

### PR DESCRIPTION
**Pull request by xinyiZzz**
1. `ClearCacheAction` support for specifying cache, `http://beip:8040/api/clear_cache/all`, `http://beip:8040/api/clear_cache/DataPageCache`
2. Optimize result returned by `ClearCacheAction`.
3. Optimize Allocator large memory printing stacktrace
4. `cache_prune_stale_thread` no longer check conf to prune cache, which has delays and brings a bad experience.

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -133,13 +133,11 @@ DEFINE_mString(process_full_gc_size, "20%");
 // If false, cancel query when the memory used exceeds exec_mem_limit, same as before.
 DEFINE_mBool(enable_query_memory_overcommit, "true");
 
-DEFINE_mBool(disable_memory_gc, "false");
+DEFINE_mBool(enable_memory_reclamation, "true");
```
Comment (yiguolei) [inline be/src/common/config.cpp:136]: 这些配置文件的名字先别改了，影响不大。另外，改了还得改文档，各个版本之间兼容还有问题。

Comment (xinyiZzz) [reply be/src/common/config.cpp]: done

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**